docs(production): (#114)
Clinical-Genomics/development: docs/github/pr-etiquette.md

As a requester:

 - [Request a review](pr-request.md) to the code owner of the tool, not a team. This should happen automatically when [code owners](https://help.github.com/en/articles/about-code-owners) are set up for the repository.
 - Reach out within the pull request or preferably face to face with the reviewers to communicate the urgency and get an estimated time to review.
 - Be specific about what you're asking from the reviewer: are you asking for a code review or to test your pull request?

As a reviewer:

>  - When you receive a pull request review and are unable to finish it within a reasonable time, you are responsible to inform that you are unable to review in a timely manner.

Reviewer: ```suggestion
 - When you accept to do a pull request review and are unable to finish it within a reasonable time, you should inform that you are unable to review in a timely manner
```
